src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/notifications/CaseNotesNotificationsService.kt

```diff
@@ -43,7 +43,7 @@ class CaseNotesNotificationsService(
     val senderIsResponsibleOfficer =
       userTypeChecker.isProbationPractitionerUser(sender) &&
         (
-          (responsibleOfficer.authUser?.let { it == sender } ?: false) ||
+          (responsibleOfficer.authUser?.let { it.authSource == sender.authSource && it.username == sender.userName } ?: false) ||
```

Comment:
this shouyld be handled in the equals method of `AuthUser`. we shouldn't duplicate it here.